### ISSUE TITLE
Option to add widget instead of wallpaper in Unlock Clock

### ISSUE THREAD

**Issue by svkrclg** [googlecreativelab/digital-wellbeing-experiments-toolkit#1]
Addition of widget will further avoid user's to compromise with there current wallpaper.
Opinion are welcomed?
Regarding "Unlock clock "

**Comment by jishhd** [comment]
Agreed, especially because wallpapers can be covered with widgets and icons themselves -- any data displayed has the chance to be covered up. A full-screen (4x4?) widget might be useful here.

**Comment by svkrclg** [comment]
So, are the PR welcomed in this repository or it's just for reference purposes?
Good to have some words from repo owner itself
